transport.go

```diff
@@ -66,7 +66,8 @@ type Transport struct {
 
 	conn rawConn
 
-	closeQueue chan closePacket
+	closeQueue          chan closePacket
+	statelessResetQueue chan sendStatelessResetArgs
```

Comment:
Probably easier: just pass the `receivedPacket` around.

Reply:
that does sound better.